app/api/v2/management/member_transfers.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module API
+  module V2
+    module Management
+      class MemberTransfers < Grape::API
+        desc 'Creates new member transfer.' do
+          @settings[:scope] = :write_transfers
+        end
+        params do
+          requires :key,
+                   type: String,
+                   desc: 'Unique Transfer Key.'
+          requires :description,
+                   type: String,
+                   desc: 'Transfer Description.'
+          requires :currency_id,
+                   type: String,
+                   values: -> { Currency.codes(bothcase: true) },
+                   desc: 'Operation currency.'
+          requires :amount,
+                   type: BigDecimal,
+                   values: ->(v) { !v.to_d.zero? },
+                   desc: 'Operation amount.'
+          requires :service,
+                   type: String,
+                   values: MemberTransfer::AVAILABLE_SERVICES
+          requires :member_uid
+        end
+        post '/member_transfers' do
+          mt = MemberTransfer.create! declared(params).merge(meta: params)
```

Comment:
Забылипро транзакцию. В результате если ошибка произойдет ниже (а она произошла - https://app.bugsnag.com/bitzlato/peatio/errors/61fad25944dd1700085a7db3?filters[event.since]=30d&filters[error.status]=open&filters[app.release_stage]=production) но мы никогда в жизги не узнаем зачислилось лионо на самом деле или нет

Fix - https://github.com/bitzlato/peatio/commit/e558e3699aaeb3803f71d6597ed451565f8ba215